modules/virt-supported-features-for-virtual-storage-matrix.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-supported-features-for-virtual-storage.adoc
+
+[id="virt-supported-features-for-virtual-storage-matrix_{context}"]
+= Supported features for {VirtProductName} virtual storage
+
+.{VirtProductName} virtual storage feature support matrix
+[cols="40%,15%,15%,15%,15%"]
+|===
+||Virtual machine live migration  |Host-assisted virtual machine disk cloning |Storage-assisted virtual machine disk cloning |Virtual machine import
+
+|{product-title} container storage: RBD block volumes
```

Comment:
RBD is block
you can state here Ceph-rbd or Ceph Block

Reply:
Actually RBD does not automatically mean block.  You can create a Filesystem PVC on RBD and k8s will create a filesystem on the block device.  We don't recommend using it that way.  Because of this I think we need to say both rbd and block.  Would it be clearer to say "RBD block-mode volumes"?

Reply:
Going with "RBD block-mode volumes"